Arduino/IR_ESP8266/IR_ESP8266.ino

```diff
@@ -0,0 +1,35 @@
+#include <Arduino.h>
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+
+const uint16_t sendPin = 1;
+bool switchFlag = false;
+
+IRsend irsend(sendPin);
+
+void setup() {
+  irsend.begin();
+
+  attachInterrupt(digitalPinToInterrupt(sendPin), triggerPull, HIGH);
+}
+
+void loop() {
+  if(switchFlag){
+    //send to transistor
+    //TODO add in an actual LG signal
+    Serial.println("NEC");
+    irsend.sendNEC(data, bits)
```

Comment:
We will get the value for data on Friday. Once we verify that the signal works, I will approve this pull request.